src/Gui/DlgAddPropertyVarSet.h

```diff
@@ -117,6 +125,7 @@ public Q_SLOTS:
     std::string namePropertyToAdd;
     std::unique_ptr<PropertyEditor::PropertyItem> propertyItem;
     std::unique_ptr<App::ObjectIdentifier> objectIdentifier;
+    bool criticalDialogShown;
```

Comment:
Did you consider making this a static local instead of a member? It's only used in that one method, right?

Reply:
Yes, that's true. I didn't consider it. Do you prefer that?

Reply:
Generally speaking I prefer to keep variables as local as possible -- it makes them easier to reason about, and it's harder to mess up (say by some later bit of code deciding to also use this variable for something). I'm not going to die on this hill, but if you want to make the change go for it.

Reply:
@chennes Switched to the local static variable.